<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ispDialog</class>
 <widget class="QDialog" name="ispDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1280</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>1280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>1280</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>-1</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">SinRobot Fast ISP</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

QDialog
{
	background-color: rgb(234, 234, 234);
};
font: 28px &quot;微软雅黑&quot;;</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutDialog">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="labelLogo">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>60</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../xPad.qrc">:/img/other/ispLogo.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="IsMainArea" stdset="0">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayoutFrame">
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>操作</string>
        </property>
        <layout class="QGridLayout" name="gridLayoutOperation">
         <property name="verticalSpacing">
          <number>4</number>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
         <item row="1" column="0" colspan="4">
          <widget class="QTableView" name="fileView">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QTableView
{
	selection-background-color: rgb(64, 168, 255);
	background-color: rgb(230, 255, 255);
	alternate-background-color: rgb(199, 255, 255);
	gridline-color: rgb(168, 180, 168);
}
</string>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="autoScroll">
            <bool>false</bool>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="tabKeyNavigation">
            <bool>false</bool>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="dragDropOverwriteMode">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="cornerButtonEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>0</number>
           </attribute>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>40</number>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>40</number>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>40</number>
           </attribute>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="xPixmapButton" name="Button_Browse">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>70</height>
            </size>
           </property>
           <property name="palette">
            <palette>
             <active>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>159</red>
                 <green>158</green>
                 <blue>158</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>刷新</string>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="imageNormal">
            <pixmap resource="../../xPad.qrc">:/img/widget/BtnBoxNormal.png</pixmap>
           </property>
           <property name="imagePress">
            <pixmap resource="../../xPad.qrc">:/img/widget/BtnBoxPress.png</pixmap>
           </property>
           <property name="borderTop">
            <number>8</number>
           </property>
           <property name="borderRight">
            <number>8</number>
           </property>
           <property name="borderBottom">
            <number>8</number>
           </property>
           <property name="borderLeft">
            <number>8</number>
           </property>
           <property name="contentAlignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_7">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>波特率：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="comboBox_baud">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>70</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>1</width>
             <height>32</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string notr="true">115200</string>
            </property>
            <property name="icon">
             <iconset resource="../../xPad.qrc">
              <normaloff>:/img/widget/TabHeader.png</normaloff>:/img/widget/TabHeader.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">230400</string>
            </property>
            <property name="icon">
             <iconset resource="../../xPad.qrc">
              <normaloff>:/img/widget/TabHeader.png</normaloff>:/img/widget/TabHeader.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">380400</string>
            </property>
            <property name="icon">
             <iconset resource="../../xPad.qrc">
              <normaloff>:/img/widget/TabHeader.png</normaloff>:/img/widget/TabHeader.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">460800</string>
            </property>
            <property name="icon">
             <iconset resource="../../xPad.qrc">
              <normaloff>:/img/widget/TabHeader.png</normaloff>:/img/widget/TabHeader.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">921600</string>
            </property>
            <property name="icon">
             <iconset resource="../../xPad.qrc">
              <normaloff>:/img/widget/TabHeader.png</normaloff>:/img/widget/TabHeader.png</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="xPushButton" name="Button_PROGRAM">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>70</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>编程</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="4">
          <widget class="QLabel" name="label_CPU">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string notr="true">CPU：</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="4">
          <widget class="QProgressBar" name="progressBar">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="textVisible">
            <bool>true</bool>
           </property>
           <property name="format">
            <string notr="true">%p%</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QLabel" name="labelVersion">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>软件版本：V2.11</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>请选择文件：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="labelMessage">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_TIME">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string notr="true">Time: 0s</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="xPixmapButton" name="BtnRestart">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>70</height>
         </size>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>159</red>
              <green>158</green>
              <blue>158</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>重启</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="imageNormal">
         <pixmap resource="../../xPad.qrc">:/img/widget/BtnBoxNormal.png</pixmap>
        </property>
        <property name="imagePress">
         <pixmap resource="../../xPad.qrc">:/img/widget/BtnBoxPress.png</pixmap>
        </property>
        <property name="borderTop">
         <number>8</number>
        </property>
        <property name="borderRight">
         <number>8</number>
        </property>
        <property name="borderBottom">
         <number>8</number>
        </property>
        <property name="borderLeft">
         <number>8</number>
        </property>
        <property name="contentAlignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>xPixmapButton</class>
   <extends>QWidget</extends>
   <header>xpixmapbutton.h</header>
  </customwidget>
  <customwidget>
   <class>xPushButton</class>
   <extends>QPushButton</extends>
   <header>xpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../xPad.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Button_PROGRAM</sender>
   <signal>clicked()</signal>
   <receiver>ispDialog</receiver>
   <slot>clickProgram()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnRestart</sender>
   <signal>clicked()</signal>
   <receiver>ispDialog</receiver>
   <slot>clickRestart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>622</x>
     <y>452</y>
    </hint>
    <hint type="destinationlabel">
     <x>617</x>
     <y>696</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_Browse</sender>
   <signal>clicked()</signal>
   <receiver>ispDialog</receiver>
   <slot>clickBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>679</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>clickProgram()</slot>
  <slot>clickBrowse()</slot>
  <slot>SaveINI_File(QString)</slot>
  <slot>SaveINI_COM(int)</slot>
  <slot>SaveINI_Fast(bool)</slot>
  <slot>clickRestart()</slot>
 </slots>
</ui>
